tests/foreman/ui/test_hardwaremodel.py

```diff
@@ -8,7 +8,7 @@
 
 :CaseComponent: Hosts
 
-:Assignee: pdragun
+:Assignee: tstrych
```

Comment:
Please don't change the assignee of these tests

Reply:
Interesting - Why there are no conflicts :)

Reply:
Probably wrong rebase on master :)

Reply:
@peterdragun yes that was a wrong rebase. I updated it.